README.md

```diff
@@ -108,7 +108,7 @@ support today's systems.
 
 ## Test builds / development snapshots
 
-[Links to the newest builds](https://dosbox-staging.github.io/downloads/devel/)
+[Links to the newest builds](https://dosbox-staging.github.io/downloads/development-builds/)
```

Comment:
Thanks @XMhat.

Can you de-dupe to two dev-build links?

I believe the `[key]:URL` notation is a hidden lookup value.

So pressumably the `Links to the newest builds` at the bottom could re-use the `[Development builds]` keyed URL (I think)?

Reply:
There you go. You learn something new every day.